src/features/Mission/Mission.jsx

```diff
@@ -23,30 +70,34 @@ const Mission = () => {
   //   }, [status, dispatch]);
 
```

Comment:
- [Optional] Kindly make sure there are no useless comments in your code, they make your code look unprofessional

Reply:
Thanks for the feedback @jonathanmunamire , I will make changes as suggested :smile: